File: examples/python/send.py
import os

from kalourmade_sailup import SailupClient

client = SailupClient(os.environ["SAILUP_API_KEY"])

message = client.sms.send(
    from_="MyBrand",
    to=["233241234567"],
    body="Hello from the Sailup Python SDK",
)

print(f"Sent {message.id} — delivery_status={message.delivery_status.value}")

File: examples/php/send.php
<?php

require __DIR__ . '/../../vendor/autoload.php';

use Kalourmade\Sailup\SailupClient;

$client = new SailupClient(getenv('SAILUP_API_KEY') ?: throw new RuntimeException('Set SAILUP_API_KEY'));

$message = $client->sms->send(
    from: 'MyBrand',
    to: ['233241234567'],
    body: 'Hello from the Sailup PHP SDK'
);

printf("Sent %s — delivery_status=%s\n", $message->id, $message->deliveryStatus->value);
